stbt/__init__.py

```diff
@@ -74,10 +79,14 @@
     "Region",
     "save_frame",
     "TextMatchResult",
+    "TransitionResult",
```

Comment:
I don't think `TransitionResult` belongs here. Users should not be constructing them.